File: python-vs-javascript/2-selection/2-if-else.js
//If, else if, else checking input
let response = prompt("Type Y(es), N(o), or A(bort)");
response = response.toLowerCase();

if (response == 'y'){
    console.log("That was a good choice");
}
else if (response == 'n') {
    console.log("That was a bad choice");
} 
else if  (response == 'a') {
    console.log("That's a bad plan") ;    
} 
else {
    console.log("Were you listening to the question as written?");
}

File: python-vs-javascript/2-selection/2-if-else.py
#If, elif, else checking input
response = input("Type Y(es), N(o), or A(bort)")
response = response.lower()

if response == 'y':
    print("That was a good choice")
    
elif response == 'n':
    print("That was a bad choice")

elif response == 'a':
    print("That's a bad plan") 
    
else:
    print("Were you listening to the question as written?")
